### ISSUE TITLE
Setup Monorepo

### ISSUE THREAD

**Issue by stephan418** [stephan418/monhike#7]
- [ ] Set up documentation project(s)
- [ ] Set up frontend project(s)
- [ ] Set up backend solution

**Comment by euclid9099** [comment]
could/should this be combined with [Setup Github](https://github.com/stephan418/monhike/issues/5)?

**Comment by stephan418** [comment]
I don't think so, this is about setting up the repository and the GH issue is about setting up protection policied and GH specific things
